lib/cousins.ml

```diff
@@ -355,15 +387,16 @@ let init_asc_cnt base max_a_l p =
         asc_cnt.(0) <-
           [ (get_iper p, [ Gwdb.dummy_ifam ], Gwdb.dummy_iper, [ 0 ]) ];
         for i = 1 to max_a_l do
-          asc_cnt.(i) <- ascendants base [] asc_cnt.(i - 1) i
+          asc_cnt.(i) <- ascendants base [] asc_cnt.(i - 1)
         done;
         asc_cnt
       in
       asc_cnt_t := Some t';
       t'
 
 (* tableau des ascendants de p *)
-let init_desc_cnt base max_d_l p =
+let init_desc_cnt conf base p =
+  let max_d_l = Util.max_descendant_level conf base (get_iper p) 120 in
```

Comment:
This '120' appears multiple times, what is it? the maximum ancestor/descendant level? maybe make it a variable